src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -93,7 +81,7 @@ object Read extends Logging {
    * @param record
    * @return
    */
-  def fromSAMRecord(record: SAMRecord): Read = {
```

Comment:
I actually kind of like the explicitness of `fromSAMRecord` and `fromADAMRecord` instead of just an overloaded `apply`, but if you feel strongly this is better i'm ok with it